bundles/org.openhab.ui.cometvisu/src/main/resources/openapi.yaml

```diff
@@ -238,7 +238,7 @@ paths:
           content:
             application/json:
               schema:
-                oneOf:
+                allOf:
```

Comment:
Is there a special reason for this change or did you just replace all oneOfs? I don't see that this change is needed, my local build works without it. If you revert this change, I would approve this PR.

Reply:
Great, thanks. I think I just replaced all of them. Actually this gave me some headache when I updated the generator.

You are right, it generates fine even without this. Reverted.